SafeExamBrowser.UserInterface.Desktop/Windows/FileSystemDialog.xaml.cs

```diff
@@ -29,7 +31,6 @@ internal partial class FileSystemDialog : Window
 		private string message;
 		private FileSystemOperation operation;
 		private IWindow parent;
-		private bool restrictNavigation;
```

Comment:
Why did you remove this flag? We need this for a use case where navigation within the dialog should be restricted to a specific folder.

Reply:
You're right, sorry I didn't see that there were changes in 3.0.1+ and overwrote them. I have reworked this part again.